typesystem/serialization/test.cc

```diff
@@ -241,6 +247,7 @@ TEST(Serialization, Binary) {
 #endif
 
 TEST(JSONSerialization, CPPTypes) {
+  using namespace serialization_test;
```

Comment:
This `using` looks unnecessary. Even if there was anything from that namespace, it would look more aligned with explicit references like in the test below: `serialization_test::Serializable`, `serialization_test::Enum`.

Reply:
That's generally how we do it in the tests, to make the test body code as clean and uncluttered of an example code as possible.